2023-05-24 14:04:49,518 ERROR 19356 [-/127.0.0.1/-/12ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:51:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19356
hostname: MacBook-Pro-2.local

2023-05-24 14:04:49,544 ERROR 19356 [-/127.0.0.1/-/2ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:51:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19356
hostname: MacBook-Pro-2.local

2023-05-24 14:04:49,656 ERROR 19356 [-/127.0.0.1/-/2ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:51:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19356
hostname: MacBook-Pro-2.local

2023-05-24 14:04:50,290 ERROR 19356 [-/127.0.0.1/-/1ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:51:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19356
hostname: MacBook-Pro-2.local

2023-05-24 14:04:52,658 ERROR 19356 [-/127.0.0.1/-/4ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:51:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19356
hostname: MacBook-Pro-2.local

2023-05-24 14:05:28,572 ERROR 19455 [-/127.0.0.1/-/15ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:49:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19455
hostname: MacBook-Pro-2.local

2023-05-24 14:05:28,742 ERROR 19455 [-/127.0.0.1/-/1ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:49:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19455
hostname: MacBook-Pro-2.local

2023-05-24 14:05:31,749 ERROR 19455 [-/127.0.0.1/-/2ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at Object.appendFileSync (node:fs:2215:6)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:49:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19455
hostname: MacBook-Pro-2.local

2023-05-24 14:06:53,710 ERROR 19542 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:06:53,710 ERROR 19542 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 19542
hostname: MacBook-Pro-2.local

2023-05-24 14:06:53,711 ERROR 19542 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 19542
hostname: MacBook-Pro-2.local

2023-05-24 14:06:53,712 ERROR 19542 [-/127.0.0.1/-/1ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 19542
hostname: MacBook-Pro-2.local

2023-05-24 14:07:52,390 ERROR 19658 [-/127.0.0.1/-/18ms POST /logs] nodejs.ENOENTError: ENOENT: no such file or directory, mkdir 'Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs'
    at Object.mkdirSync (node:fs:1334:3)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:41:10)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
errno: -2
syscall: "mkdir"
code: "ENOENT"
path: "Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ENOENTError"
pid: 19658
hostname: MacBook-Pro-2.local

2023-05-24 14:09:01,717 ERROR 19750 [-/127.0.0.1/-/26ms POST /logs] nodejs.TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Object
    at Object.writeFileSync (node:fs:2146:5)
    at LogsController.writeLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:49:8)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
code: "ERR_INVALID_ARG_TYPE"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
pid: 19750
hostname: MacBook-Pro-2.local

2023-05-24 14:18:25,514 ERROR 20588 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:18:25,514 ERROR 20588 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20588
hostname: MacBook-Pro-2.local

2023-05-24 14:18:25,515 ERROR 20588 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20588
hostname: MacBook-Pro-2.local

2023-05-24 14:18:25,516 ERROR 20588 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20588
hostname: MacBook-Pro-2.local

2023-05-24 14:20:00,430 ERROR 20713 [-/127.0.0.1/-/10ms POST /readlogs] nodejs.ENOENTError: ENOENT: no such file or directory, open '/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/2023-05-24.log'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at LogsController.readLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:62:24)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
errno: -2
syscall: "open"
code: "ENOENT"
path: "/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/2023-05-24.log"
name: "ENOENTError"
pid: 20713
hostname: MacBook-Pro-2.local

2023-05-24 14:21:09,609 ERROR 20832 [-/127.0.0.1/-/5ms POST /readlogs] nodejs.ReferenceError: ctx is not defined
    at LogsController.readLogs (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/controller/logs.js:64:5)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:45:18)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:186:18
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:44:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/node_modules/koa-compose/index.js:36:12
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/@eggjs/router/lib/router.js:191:33)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at paramsErr (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/middleware/paramsErr.js:19:11)
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/koa-compose/index.js:42:32)

pid: 20832
hostname: MacBook-Pro-2.local

2023-05-24 14:21:54,327 ERROR 20886 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:21:54,327 ERROR 20886 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20886
hostname: MacBook-Pro-2.local

2023-05-24 14:21:54,328 ERROR 20886 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20886
hostname: MacBook-Pro-2.local

2023-05-24 14:21:54,329 ERROR 20886 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20886
hostname: MacBook-Pro-2.local

2023-05-24 14:22:38,323 ERROR 20949 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:22:38,324 ERROR 20949 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20949
hostname: MacBook-Pro-2.local

2023-05-24 14:22:38,326 ERROR 20949 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20949
hostname: MacBook-Pro-2.local

2023-05-24 14:22:38,327 ERROR 20949 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 20949
hostname: MacBook-Pro-2.local

2023-05-24 14:35:37,590 ERROR 21838 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:35:37,591 ERROR 21838 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 21838
hostname: MacBook-Pro-2.local

2023-05-24 14:35:37,592 ERROR 21838 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 21838
hostname: MacBook-Pro-2.local

2023-05-24 14:35:37,593 ERROR 21838 [-/127.0.0.1/-/1ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 21838
hostname: MacBook-Pro-2.local

2023-05-24 14:39:42,656 ERROR 22073 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 14:39:42,657 ERROR 22073 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 22073
hostname: MacBook-Pro-2.local

2023-05-24 14:39:42,665 ERROR 22073 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 22073
hostname: MacBook-Pro-2.local

2023-05-24 14:39:42,671 ERROR 22073 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 22073
hostname: MacBook-Pro-2.local

2023-05-24 15:43:55,357 ERROR 26076 [egg-mongoose] mongodb://43.134.241.119:8082/webgps disconnected
2023-05-24 15:43:55,359 ERROR 26076 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 26076
hostname: MacBook-Pro-2.local

2023-05-24 15:43:55,359 ERROR 26076 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 26076
hostname: MacBook-Pro-2.local

2023-05-24 15:43:55,362 ERROR 26076 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [43.134.241.119:8082] on first connect [MongoNetworkTimeoutError: connection timed out
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at connectionFailureError (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:362:14)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:16)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)]
    at Pool.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/topologies/server.js:441:11)
    at Pool.emit (node:events:390:28)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:564:14
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/pool.js:1000:11
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:32:7
    at callback (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:300:5)
    at Socket.<anonymous> (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/mongodb/lib/core/connection/connect.js:330:7)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:486:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
name: "MongoNetworkError"
pid: 26076
hostname: MacBook-Pro-2.local

2023-05-24 17:13:29,212 ERROR 32240 [-/127.0.0.1/-/0ms GET /] nodejs.ReferenceError: Obj is not defined
    at module.exports (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/app/model/logs.js:61:13)
    at getExports (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/file_loader.js:220:15)
    at FileLoader.parse (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/file_loader.js:158:25)
    at FileLoader.load (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/file_loader.js:67:24)
    at AppWorkerLoader.loadToApp (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/loader/egg_loader.js:399:25)
    at loadModelToApp (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-mongoose/lib/mongoose.js:124:14)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-mongoose/lib/mongoose.js:53:7
    at Object.callFn (/Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/utils/index.js:44:42)
    at /Users/lovewcc/Documents/Study/全栈-前后端导航项目/vue-webstack-server/node_modules/egg-core/lib/lifecycle.js:265:13
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

pid: 32240
hostname: MacBook-Pro-2.local

